Gui/opensim/view/src/org/opensim/view/experimentaldata/ExperimentalMarker.java

```diff
@@ -104,9 +107,11 @@ public JSONObject createDecorationJson(ArrayList<UUID> comp_uuids, MotionDisplay
         StateVector dataAtStartTime = motionDisplayer.getSimmMotionData().getStateVector(0);
         ArrayDouble interpolatedStates = dataAtStartTime.getData();
         int idx = getStartIndexInFileNotIncludingTime();
+        AnnotatedMotion mot = (AnnotatedMotion)motionDisplayer.getSimmMotionData();
+        conversion = mot.getUnitConversion();
         JSONArray pos = new JSONArray();
         for (int i = 0; i < 3; i++) {
-            pos.add(interpolatedStates.get(idx+i));
+            pos.add(interpolatedStates.get(idx+i)/conversion);
```

Comment:
I agree that on the core side we use multiplication, and I could invert the meaning of Scale factor but I prefer to make the change as surgical as possible at this point to limit scope.
I can add a comment somewhere to indicate the plan so the GUI and core use similar conventions.

Reply:
At this point I'd prefer not to make a wholesale changes especially when we're planning to rewrite but I'd be happy to adding documentation to  help maintainability over the short term.

Reply:
I don't know what you mean by "wholesale changes". I am not asking for any rewrite. I am simply asking for:
1. do not invert the unit conversion on ln 58 of FileLoadDataAction.java as I indicated above
2. on ln 118 multiply by `conversion` instead of divided.

Simple changes.

Reply:
@aseth1 I prefer to make the changes only local to htis file as the issue was specific to trc previewing. Conversion is used in other places  to compute bounding boxes for example and/or   force visualization so I don't plan to touch at this point. 
Making the change here would make it inconsistent with other places in the GUI code base but I'll do it if you think it helps.